kafka/producer/base.py

```diff
@@ -27,6 +27,9 @@
 
 STOP_ASYNC_PRODUCER = -1
 
+# unlimited
+ASYNC_QUEUE_MAXSIZE = 0
```

Comment:
I don't want to change the default behavior of the library. By default is ``unlimited``, but you have the choice of setting up a limit. Does it make sense?